cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -603,6 +606,8 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
                                 clusterCaConfig != null ? clusterCaConfig.getCertificateExpirationPolicy() : null);
                         clusterCa.createRenewOrReplace(
                                 reconciliation.namespace(), reconciliation.name(), caLabels.toMap(),
+                                kafkaClusterTemplate != null && kafkaClusterTemplate.getClusterCaCert() != null && kafkaClusterTemplate.getClusterCaCert().getMetadata() != null
```

Comment:
Probably marginally better for readability if we move this ternary before we pass to the method call?
i.e. make it an actual if check.

Reply:
I create a variable, but maybe it is better just a tab. What do you think?